Jenkinsfile

```diff
@@ -1,6 +1,22 @@
 pipeline {
     agent none
     stages {
+        stage('Check formatting and Clippy') {
+            environment {
+                CARGO_HOME = '/home/jenkins/.cargo'
+                RUSTUP_HOME = '/home/jenkins/.rustup'
+                RUSTFLAGS = "-D warnings"
+            }
+            agent {
+                label 'linux'
+            }
+            steps {
+                echo 'Checking formatting...'
+                sh '$CARGO_HOME/bin/cargo fmt -- --check'
```

Comment:
is it possible to break these out into different steps? I find it easier to parse the issues then.

Reply:
Those are different steps. Each command is a step.

Reply:
Oh, if you mean stage, sure.